### DISCUSSION TITLE
University of Illinois System Shared GitHub Service

### DISCUSSION THREAD

**Discussion by Bashima** [uillinois-community/uillinois-community.github.io#21] [Q&A] [answered]
Can we change one of the organization owner after creating the GitHub for the organization? Also can we use institution (e.g., Bachman Institution), instead of a department?

**Comment by areynold** [comment, marked as answer]
The org owner can be changed in the github org's settings. Go to your organization's landing page and click the People tab. Find the person who should be made owner, click the Settings menu in the Role column next to their name, and choose a new role (Owner in this case). You can then remove your old owner from the org or assign them a less privileged role.
